### PR TITLE
Push/Pull In Hell

### PR DESCRIPTION
**Pull request by TrystanOakley**
@sstangl this one suuuuuucked....and it was just a push/pull -_- Oh well

### CODE REVIEW

File: meet-data/spf-archive/0447/lifters.csv [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Name,Division,BodyweightKg,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Equipment,Event,Sex,WeightClassKg,Place
+Michael Walker,Pre-teen,44,,13.61,36.29,49.9,Raw,BD,M,44,1
+Tanner Ezell,Teenage (16-17),67.5,,113.4,183.7,297.1,Raw,BD,M,67.5,1
```
Comment (sstangl) [inline meet-data/spf-archive/0447/lifters.csv]: Tanner's 430lb bench seems to have been dropped.

File: meet-data/spf-archive/0447/lifters.csv [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Name,Division,BodyweightKg,BestSquatKg,BestBenchKg,BestDeadliftKg,TotalKg,Equipment,Event,Sex,WeightClassKg,Place
+Michael Walker,Pre-teen,44,,13.61,36.29,49.9,Raw,BD,M,44,1
```
Comment (sstangl) [inline meet-data/spf-archive/0447/lifters.csv]: Michael's 85lb bench attempt was dropped. You can create a column "Bench4LBS" and then use the normal csv-tokg script.

File: meet-data/spf-archive/0447/lifters.csv [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+Tanner Ezell,Teenage (16-17),67.5,,113.4,183.7,297.1,Raw,BD,M,67.5,1
+Clayton Harrell,Teenage (18-19),82.5,,106.59,197.31,303.91,Raw,BD,M,82.5,1
+Trystan Oakley,Teenage (18-19),90,,147.42,231.33,378.75,Raw,BD,M,90,1
```
Comment (sstangl) [inline meet-data/spf-archive/0447/lifters.csv]: :-)

**Comment by TrystanOakley** [comment]
I deleted 4th attempts. Thought I wasn't supposed to have them. My bad.

File: meet-data/spf-archive/0447/lifters.csv (hunk 3 above)
Comment (TrystanOakley) [inline meet-data/spf-archive/0447/lifters.csv]: Yeah lol little ol me back in 2015

File: meet-data/spf-archive/0447/lifters.csv (hunk 1 above)
Comment (TrystanOakley) [inline meet-data/spf-archive/0447/lifters.csv]: thats a DL 4th attempt. Fixing now

File: meet-data/spf-archive/0447/lifters.csv (hunk 2 above)
Comment (TrystanOakley) [inline meet-data/spf-archive/0447/lifters.csv]: Its a DL 4th attempt. Fixing now.

**Comment by sstangl** [comment]
Phew. Thanks!

**Pull request merged**